### ISSUE TITLE
apollo-cache-inmemory: the new release seems to break the build

### ISSUE THREAD

**Issue by surbina** [apollographql/apollo-client#4332]
**Intended outcome:**
Installing apollo-cache-inmemory and using it in a project should work without problems.

**Actual outcome:**
Installing apollo-cache-inmemory and using it in a project breaks and shows the following error:
`Module not found: Error: Can't resolve 'tslib'`

It seems that the compiled version of apollo-cache-inmemory includes an import of tslib. But tslib does not appear in the apollo-cache-inmemory dependencies.

<img width="570" alt="screen shot 2019-01-18 at 5 31 18 pm" src="https://user-images.githubusercontent.com/7356880/51411531-f561d180-1b46-11e9-961d-d927a388c7c2.png">

**How to reproduce the issue:**
Install apollo-cache-inmemory in a project where you are not using TypeScript and build the project using Webpack.

**Versions**
  System:
    OS: macOS High Sierra 10.13.6
  Binaries:
    Node: 8.5.0 - ~/.nvm/versions/node/v8.5.0/bin/node
    Yarn: 1.6.0 - ~/.nvm/versions/node/v8.5.0/bin/yarn
    npm: 5.8.0 - ~/.nvm/versions/node/v8.5.0/bin/npm
  Browsers:
    Chrome: 71.0.3578.98
    Firefox: 60.0
    Safari: 12.0.2

**Comment by benjamn** [comment]
Thanks for reporting this!

The idea was supposed to be that the client packages would import `tslib` from the root `apollo-client` project directory, and Rollup would inline the imports at build time, so there would be no runtime import from `tslib`. This was important to make sure we only imported the `tslib` helpers that we actually used, instead of depending on the whole library.

Not sure yet why that isn't working as intended, but I will find out!
